octoprint_virtual_printer_settings/__init__.py

```diff
@@ -11,26 +11,29 @@
 
 import octoprint.plugin
 
-class VirtualPrinterSettingsPlugin(octoprint.plugin.TemplatePlugin):
+
+class VirtualPrinterSettingsPlugin(octoprint.plugin.TemplatePlugin, octoprint.plugin.AssetPlugin):
 
 	def get_template_configs(self):
 		return [dict(
-			type='settings',
-			name='Virtual Printer',
-			template='virtual_printer_new_settings.jinja2',
-			replaces='plugin_virtual_printer',
+			type="settings",
+			name="Virtual Printer",
+			replaces="plugin_virtual_printer",
+			template="virtual_printer2_settings.jinja2",
 			custom_bindings=False
 		)]
 
+	def get_assets(self):
+		return dict(
+			js=["js/jquery-ui.min.js", "js/knockout-sortable.js", "js/virtual_printer_settings.js"]
```

Comment:
Are these part of developing the lists? Doesn't looked like they've been added to the PR

Reply:
oh yeah, did I not push those...let me fix that for you.